runtime/app/xwalk_main_delegate.cc

```diff
@@ -76,6 +77,10 @@ void XWalkMainDelegate::PreSandboxStartup() {
       command_line->GetSwitchValueASCII(switches::kProcessType);
   InitLogging(process_type);
 #endif
+
+#if !defined(OS_ANDROID)
```

Comment:
I dont'see this ifdef in Chromium code; what happens if this function is called on Android?